crates/y-sweet/src/main.rs

```diff
@@ -51,12 +51,18 @@ enum ServSubcommand {
 
 fn get_store_from_opts(store_path: &str) -> Result<Box<dyn Store>> {
     if store_path.starts_with("s3://") {
-        let region = Region::UsEast1;
-
+        let region = match Region::from_default_env() {
+            Ok(region) => {
+                tracing::info!(region=?region, "Using region from environment.");
+                region
+            }
+            Err(e) => {
+                tracing::warn!(error=?e, "Failed to get region from environment, using default.");
```

Comment:
Worth mentioning here that the default value is us-east-1?

Reply:
Done.